pkg/scheduler/apis/config/v1/defaults.go

```diff
@@ -0,0 +1,251 @@
+/*
```

Comment:
It looks like there were no changes in effective default values from v1beta3 → v1. Once we release v1, the defaults for it are locked, so were there any TODOs or outstanding issues where a tweak to defaults was desired?

Reply:
The last iteration was v1beta3, introduced in 1.23, which increased some plugin weights https://github.com/kubernetes/enhancements/tree/master/keps/sig-scheduling/785-scheduler-component-config-api#proposal

Are we sure this is enough?

@Huang-Wei @kerthcet @damemi @denkensk for thoughts. Holding the graduation to 1.26 is an option.

Reply:
My question is if we want to change the plugins and weights(maybe we treat these two things differently) in the future, should we upgrade the CC again, like v1 -> v2alpha1? If so, being cautious is not a bad thing. Maybe we need some feedbacks like a survey.

If no, since I have not gotten any negative feedbacks from the community so far(maybe just missed). From the guides in `Graduation Criteria`,  it seems ready to GA now. 
```
Deprecation of legacy plugins.
Minimal changes in the last beta iteration of the API.
```

Reply:
I would prefer not to start a v2 right when we just released v1 :)

I actually expect that most of the world is still running k8s 1.22 or older. Or even if running 1.23, some might still using v1beta2. Since there is no pressure to get this to GA, maybe it's not a bad idea to send a survey to sig-scheduling? The focus of the survey should be on whether user-influenced like pod topology spreading give strong signal compared to internal scores.

Unless people really feel strong about graduating to GA and leaving it up to providers to decide if they want to change the weights for their customers.

Reply:
It seems we came to the consequence to graduate this to GA.